resolwe_bio/processes/expression/cuffnorm.yml

```diff
@@ -7,8 +7,10 @@
   name: Cuffnorm
   requirements:
     expression-engine: jinja
+    resources:
+      cores: 1    
   data_name: 'Cuffnorm ({{cuffquant|map("sample_name")|join(", ")|default("?")}})'
-  version: 1.1.1
+  version: 1.1.2
```

Comment:
Change the version to `1.2.0` as output is not the same as in the previous version.

Reply:
The version was changed due to core requirement change but I was hesitant of changing the version to 1.2.0 later since the process itself didn't change, only a tool.

Reply:
@mstajdohar @kostko 
Should the version be changed if the .yaml file was not changed at all but only a tool it uses? If yes, that would mean that use of a newer version of a package in docker image would result in having to bump all processes (that use that package) versions.

Reply:
This is not required, but it is considered a good practice to also bump the process version.